core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java

```diff
@@ -88,7 +88,7 @@
      */
     public static final String REQUEST_POST_METHOD = "POST";
 
-    public static final String MULTIPART_FORM_DATA_REGEX = "^multipart\\/form-data(; boundary=[\\-a-zA-Z0-9]{1,70})?";
+    public static final String MULTIPART_FORM_DATA_REGEX = "^multipart/form-data(; boundary=[0-9a-zA-Z'()+_,\\-./:=?]{1,70})?";
```

Comment:
There are two test cases that you can extend or another one - see `DispatcherTest`